tests/sample/test_ebpf.c

```diff
@@ -11,16 +11,28 @@
 ebpf_map_definition_t test_map = {.size = sizeof(ebpf_map_definition_t),
                                   .type = BPF_MAP_TYPE_ARRAY,
                                   .key_size = sizeof(uint32_t),
-                                  .value_size = sizeof(uint64_t),
-                                  .max_entries = 128};
+                                  .value_size = 32,
+                                  .max_entries = 2};
 
 #pragma clang section text = "test_ext"
 int
 test_program_entry(test_program_context_t* context)
 {
+    uint32_t keys[2] = {0, 1};
+    uint8_t* values[2] = {0};
+    values[0] = bpf_map_lookup_elem(&test_map, &keys[0]);
+    values[1] = bpf_map_lookup_elem(&test_map, &keys[1]);
     test_ebpf_extension_helper_function3(0);
     if (context->data_end > context->data_start) {
+        int position;
+        int result;
         test_ebpf_extension_helper_function2(context->data_start, context->data_end - context->data_start);
+        if (values[0])
+            position =
+                test_ebpf_extension_find(context->data_start, context->data_end - context->data_start, values[0], 32);
```

Comment:
the 32 here has to match the 32 in line 14 and 35.  Don't hard code the same number multiple times.  Use a `#define` or `const` variable.